pype/plugins/global/publish/extract_burnin.py

```diff
@@ -193,6 +193,7 @@ def process(self, instance):
             self.log.debug("Output: {}".format(output))
 
             repre_update = {
+                "anatomy_template": "render",
```

Comment:
damn this will most likely break burnin paths in other sites that original client where this was made. @antirotor 
are you sure this is what is running there now?

Or rather we have to update all render templates in anatomy to match the one where this is used. The key point is that we can't be hardcoding `render` instead of `{family}` into it. So this needs to be supported by another PR in config

Reply:
This is addressed in pypeclub/pype-config#20